structuredefinitions/Extension-CareConnect-VaccinationProcedure-1.xml

```diff
@@ -237,7 +48,7 @@
         <code value="CodeableConcept" />
       </type>
       <binding>
-        <strength value="required" />
+        <strength value="extensible" />
```

Comment:
In UKCore, we've gone with Preferred as the strength for SNOMED CT valuesets - might be worth considering that in CareConnect?